doc/design/garbage-collection-uploads-asset-blobs-2.md

```diff
@@ -0,0 +1,43 @@
+# Upload and Asset Blob Garbage Collection
+
+This document presents a design for garbage collection of uploads and asset blobs in the context of the S3 trailing delete feature. It explains the need for garbage collection and describes the scenarios of orphaned uploads and orphaned asset blobs. The implementation involves introducing a new daily task to query and delete uploads and asset blobs that meet certain criteria. The document also mentions the recoverability of uploaded data and provides a GitHub branch for the implementation.
```

Comment:
Please add the GitHub branch to this doc.

Reply:
Added https://github.com/dandi/dandi-archive/pull/2068/commits/f587c4235e9a41374433b1e05750fdd8a5488f72